packages/components/src/components/calendar/calendar-body.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Views from './views';
+import { CommonPropTypes } from './views/types';
+
+type TBodyProps = CommonPropTypes & {
```

Comment:
```suggestion
type TBody = CommonPropTypes & {
```

Reply:
@niloofar-deriv, might I ask why you're suggesting this, because I seem to remember we always prepended `Props` to all the `types` when we began migrating to `ts` did the styling change? Otherwise we might end up in the same situation like `() => void` vs `VoidFunction` 😅

Reply:
AFAIK we are using `TComponentName` format for naming our types, because of that I suggested these :grin: